chyt/results/yt.192GB_YC.json

```diff
@@ -0,0 +1,60 @@
+
+{
+    "system": "CHYT",
+    "date": "2024-09-16",
+    "machine": "192GB",
```

Comment:
I am confused. L. 6 says "serverless" which typically means the results were measured in a database-as-a-service offering (such as ClickHouse Cloud). Was that the case?

If not, it would be good to specify the exact machine specs for reproducibility, see e.g. duckdb/results/c5.4xlarge.json.

Reply:
For CHYT with 48, 96 and 192 GB we use 1, 2 and 4 instances with 12 vCPU and 48 Gb RAM
For CHYT with 360 and 720 GB  -- 9 and 18 instances with 10 vCPU and 40 Gb RAM

You only configure the count and size of instances and then YTsaurus will schedule them across computational nodes of cluster.